File: csharp/armstrong-numbers/ArmstrongNumbers.cs
using System;
using System.Collections.Generic;

public static class ArmstrongNumbers
{
    public static bool IsArmstrongNumber(int number)
    {
        string numberString = number.ToString();
        double sum = 0;

        foreach (char n in numberString)
        {
            sum += (int)Math.Pow((int)char.GetNumericValue(n), numberString.Length);
        }

        return sum == number;
    }
}

File: javascript/armstrong-numbers/armstrong-numbers.js

export const isArmstrongNumber = (num) => {
  let numString = num.toString();
  let power = numString.length;
  let sum = 0;

  for (let i = 0; i < power; i++) {
    sum += Math.pow(Number(numString[i]), power);
  }

  return num === sum;
};
